haproxy/README.md

```diff
@@ -56,10 +56,12 @@ $ docker run -it --rm --name haproxy-syntax-check haproxy:1.5 haproxy -c -f /usr
 $ docker run -d --name my-running-haproxy my-haproxy
 ```
 
+You may need to publish the ports your haproxy is listening on to the host by specifying the `-p` option, for example `-p 80:80` to publish port 80. Make sure the port you're using is free.
```

Comment:
It'd probably be good if we used something like `-p 8080:80` instead of just `80:80` so it's clear that the ports do not have to be the same (it's a small change that IMO makes a huge difference in understanding).  ie:

> You may need to publish the ports your haproxy is listening on to the host by specifying the `-p` option, for example `-p 8080:80` to publish port 8080 from the container host to port 80 in the container. Make sure the port you're using is free.